samples/README.md

```diff
@@ -0,0 +1,28 @@
+CORS Sample
+===
+This sample consists of a request origin (SampleOrigin) and a request destination (SampleDestination). 
+Both have different domain names, to simulate a CORS request. 
+
+Modify Hosts File
+Windows:
+Run a text editor (e.g. Notepad) as an Administrator. Open the hosts file on the path: "C:\Windows\System32\drivers\etc\hosts".
+
+Linux:
+On a Terminal window, type "sudo nano /etc/hosts" and enter your admin password when prompted.
+
+In the hosts file, add the following to the bottom of the file:
+127.0.0.1	destination.example.com
+127.0.0.1	origin.example.com
+
+Save the file and close it. Then clear your browser history.
+
+Run the sample
+*In a command prompt window, open the directory where you cloned the repository, and open the SampleDestination directory. Run the command: dotnet run
+*Repeat the above step in the SampleOrigin directory.
+*Open a browser window and go to http://origin.example.com
```

Comment:
It would be good to add here the info about which port each of the apps will use by default